dasar/html/pengenalan-html.md

```diff
@@ -1,22 +1,96 @@
 # Pengenalan HTML
 
-## 1. Penjelasan HTML
+## 1. Permasalahan
+Bagi seorang pemula, untuk membuat website mungkin masih belum memahami struktur website dengan baik. Membuat website berdasarkan struktur website perlu dilakukan karena dapat membantu ketika mengatur komponen dalam website.  Untuk itu, langkah pertama sebelum membuat website adalah menentukan struktur dari website yang ingin kita buat seperti bagian header, body, content, menu, footer, dan lain-lain.
```

Comment:
Penjelasan awalnya kurang to the point, mending langsung dijelasin aja kalau website terdiri dari beberapa bagian

```suggestion
Website merupakan kumpulan dari beberapa halaman yang dapat diakses melalui internet. Dimana pada masing - masing halaman terdiri dari beberapa bagian seperti header, body, content, dan footer. Apabila bagian - bagian tersebut digabung, akan membentuk sebuah struktur website yang utuh.  Maka dari itu, langkah pertama untuk membuat website adalah menentukan struktur dari website yang ingin kita buat.
```